controllers/v2/unavailability/volunteer_unavailability_v2.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from flask import Blueprint, request, jsonify
+from flask_restful import reqparse, Resource, fields, marshal_with, Api, inputs
+
+import domain.base
+from domain import session_scope
+from repository.unavailability_repository import *
+
+from services.jwk import requires_auth
+
+edit_parser = reqparse.RequestParser()
+edit_parser.add_argument("userId", type=int, required=True)
+edit_parser.add_argument("eventId", type=int, required=True)
+edit_parser.add_argument("title", type=str)
+edit_parser.add_argument("start", type=inputs.datetime_from_iso8601)
+edit_parser.add_argument("end", type=inputs.datetime_from_iso8601)
+edit_parser.add_argument("periodicity", type=int)
+
+
+class volunteer_unavailability(Resource):
+    def patch(self):
+        args = edit_parser.parse_args()
+        with session_scope() as session:
+            success = edit_event(session, **args)
```

Comment:
Surely you should check the event exists before attempting to edit it? If it doesn't return a 400.